### PR TITLE
Change uses of address book in user messages

### PR DESCRIPTION
**Pull request by drustanyjt**
There should be no more appearances of address book in error messages. In accordance with nus-cs2103-AY2324S2/forum#809

### CODE REVIEW

File: src/main/java/seedu/address/MainApp.java [hunk 1]
```diff
@@ -81,12 +81,12 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
             addressBookOptional = storage.readAddressBook();
             if (!addressBookOptional.isPresent()) {
                 logger.info("Creating a new data file " + storage.getAddressBookFilePath()
-                        + " populated with a sample AddressBook.");
+                        + " populated with a sample ClinicMate.");
```
Comment (tengcharmaine) [inline src/main/java/seedu/address/MainApp.java]: maybe change it to 'a sample version of ClinicMate'? the one u have sounds a little weird.

File: src/main/java/seedu/address/MainApp.java [hunk 2]
```diff
@@ -81,12 +81,12 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
...
             }
             initialData = addressBookOptional.orElseGet(SampleDataUtil::getSampleAddressBook);
         } catch (DataLoadingException e) {
             logger.warning("Data file at " + storage.getAddressBookFilePath() + " could not be loaded."
-                    + " Will be starting with an empty AddressBook.");
+                    + " Will be starting with an empty ClinicMate.");
```
Comment (tengcharmaine) [inline src/main/java/seedu/address/MainApp.java]: same as above

**Review by tengcharmaine — COMMENTED** [review]
I think the first 2 sounds a little weird? Can consider my suggestions else the rest look okay!

**Review by jiahui0309 — APPROVED** [review]
LGTM!

**Pull request merged**